src/node_http2.cc

```diff
@@ -283,13 +251,13 @@ Http2Session::Http2Settings::~Http2Settings() {
 // Generates a Buffer that contains the serialized payload of a SETTINGS
 // frame. This can be used, for instance, to create the Base64-encoded
 // content of an Http2-Settings header field.
-inline Local<Value> Http2Session::Http2Settings::Pack() {
+Local<Value> Http2Session::Http2Settings::Pack() {
```

Comment:
is it ok to remove this `inline` directive here and in the other functions? Wouldn't this impact perf somehow?

Reply:
@mcollina It’s not only okay, it’s more or less the correct thing to do.

[This](http://en.cppreference.com/w/cpp/language/inline) has a pretty good explanation. *tl;dr*: Compilers are generally better at judging what functions to inline or not than humans, and will gladly ignore this advice. However, `inline` does affect the linkage properties of a function, and so its primary use should be to mark functions that are defined in header files (outside of a class definition).

I just tested it with gcc, and the compiler output is the same before and after this commit, byte-for-byte.